Tests/Realm.Tests/Sync/SyncTestHelpers.cs

```diff
@@ -132,5 +162,103 @@ public static string[] ExtractBaasSettings(string[] args)
 
             return tcs.Task;
         }
+
+        private static AppConfigType GetConfigTypeForId(string appName)
+        {
+            if (appName == "dotnet-integration-tests")
+            {
+                return AppConfigType.Default;
+            }
+
+            if (appName == "int-partition-key")
+            {
+                return AppConfigType.IntPartitionKey;
+            }
+
+            if (appName == "objectid-partition-key")
+            {
+                return AppConfigType.ObjectIdPartitionKey;
+            }
+
+            if (appName == "uuid-partition-key")
+            {
+                return AppConfigType.UUIDPartitionKey;
+            }
+
+            throw new NotSupportedException($"Unexpected appName: {appName}");
+        }
+
+        private class BaasClient : IDisposable
```

Comment:
This is a super barebones helper that calls into the admin API and reads all the apps rather than have them passed in as config parameters. It's intentionally not overengineered/safe because it's not intended to be a general-purpose admin api client.